AirCasting/Dashboard/AirSectionPickerView.swift

```diff
@@ -14,14 +15,16 @@ struct AirSectionPickerView: View {
                     ForEach(SelectedSection.allCases, id: \.self) { section in
                         Button(section.localizedString) {
                             withAnimation(.easeInOut(duration: 0.1)) {
-                                if section == .mobileDormant {
+                                if section == .mobileDormant || section == .fixed {
```

Comment:
Why? :P We don't need that, do we?

Reply:
You do, because without it for now: when you are at the leading edge (Following) and try to tap onto fixed but without moving the AirSection at all the 'Fixed' text will be cuted off

Reply:
oh, okey, didn't try that edgy edge case :D 
Nice!